functions.mk

```diff
@@ -2,7 +2,9 @@
 # Common functions used by Makefile & tests/Makefile
 
 define compile
-        @$(CC) -MM -MP -MT $@ -MT $(@:.o=.d) $(CFLAGS) $< > $(@:.o=.d)
+	$(ifeq ($(MACOS_UNIVERSAL),no),
```

Comment:
can you fix the indentation:)

Reply:
What kind of fix are you looking for?

Reply:
You are correct, the editor was configured for Go and missed that. Will take care of it :-)